script.js

```diff
@@ -1,6 +1,28 @@
 const toggleButton = document.getElementsByClassName('toggle-button')[0];
 const navBarLinks = document.getElementsByClassName('menu-links')[0];
+const dimBackground = document.getElementsByClassName('dim-background')[0];
+const exitMobileButton = document.getElementsByClassName('exit-mobile-menu')[0];
+const menuTabs = document.getElementsByClassName('menutabs');
 
 toggleButton.addEventListener('click', () => {
-  navBarLinks.classList.toggle('active')
-})
\ No newline at end of file
+  navBarLinks.classList.toggle('active');
+  dimBackground.classList.toggle('active');
+  toggleButton.classList.toggle('active');
+  exitMobileButton.classList.toggle('active');
+});
```

Comment:
- [ Optional] You've done really well on the project. However, I kindly and strongly suggest you implement two other features which the previous reviewer missed. Those features were put in place to help you build a certain skill set. Please do attend it 👍 

## Instructions:
![image](https://user-images.githubusercontent.com/50990925/223288617-f279c910-a4b5-4eec-8a35-e7fd81a3f2d3.png)